src/libraries/Microsoft.Extensions.Logging/src/LoggerFactoryScopeProvider.cs

```diff
@@ -54,13 +54,13 @@ void Report(Scope current)
                     {
                         // As TagObjects is a IEnumerable<KeyValuePair<string, object?>> this can be used directly as a scope.
                         // We do this to safe the allocation of a wrapper object.
-                         callback(activity.TagObjects, state);
+                        callback(activity.TagObjects, state);
                     }
 
                     if ((_activityTrackingOption & ActivityTrackingOptions.Baggage) != 0)
                     {
                         // Only access activity.Baggage as every call leads to an allocation
-                        IEnumerable<KeyValuePair<string, string?>> baggage = activity.Baggage;
```

Comment:
>?>> [](http://example.com/codeflow?start=63&length=3)

why we removed the nullability here? Activity.Baggage is already defined as nullable I guess [https://github.com/dotnet/runtime/blob/69b5d67d9418d672609aa6e2c418a3d4ae00ad18/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs#L323](https://github.com/dotnet/runtime/blob/69b5d67d9418d672609aa6e2c418a3d4ae00ad18/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs#L323)

Reply:
Because the rest of the library doesn't have nullability enabled. And since we don't, the compiler is warning. These changes will likely be undone when we enable nullability in this library.